corehq/apps/domain/tests/test_delete_domain.py

```diff
@@ -77,13 +88,27 @@ def _create_data(self, domain_name, i):
         )
         MobileBackendInvitation.objects.create(domain=domain_name, backend=backend)
 
+    @classmethod
+    def setUpClass(cls):
+        super(TestDeleteDomain, cls).setUpClass()
+        generator.instantiate_accounting()
+
     def setUp(self):
+        super(TestDeleteDomain, self).setUp()
         self.domain = Domain(name="test", is_active=True)
         self.domain.save()
         self.domain.convert_to_commtrack()
+        self.current_subscription = Subscription.new_domain_subscription(
+            BillingAccount.get_or_create_account_by_domain(self.domain.name, created_by='tests')[0],
+            self.domain.name,
+            DefaultProductPlan.get_default_plan_version(SoftwarePlanEdition.ADVANCED),
+            date_start=date.today() - relativedelta(days=1),
+        )
+
```

Comment:
If this subscription is only used in the one test, could you add it in to the test method?  Not sure if it is a costly setup, but it'd be nice to avoid calling unnecessarily.